tensorflow_probability/examples/jupyter_notebooks/Cox_Process_TFP.ipynb

```diff
@@ -0,0 +1,1409 @@
+            "124                   21                   29\n",
+            "125                    2                    4\n",
+            "126                    0                    0\n",
+            "127                    0                    0\n",
+            "128                    0                    0\n",
+            "129                    0                    4\n",
+            "130                    0                    0\n",
+            "131                   50                   39\n",
+            "132                    0                    0\n",
+            "133                   48                   38\n",
+            "134                    8                    4\n",
+            "135                    0                    0\n",
+            "136                   23                   21\n",
+            "137                    2                    0\n",
+            "138                    9                    6\n",
+            "139                   18                   17\n",
+            "140                    0                    0\n",
+            "141                    0                    0\n",
+            "142                    0                    0\n",
+            "143                    0                    0\n",
+            "144                    0                    0\n",
+            "145                    0                    0\n",
+            "146                    0                    0\n",
+            "147                    4                    6\n",
+            "148                    0                    0\n",
+            "149                    1                    4\n",
+            "150                    7                   18\n",
+            "151                    0                    0\n",
+            "152                    6                    4\n",
+            "153                    0                    0\n",
+            "154                    4                    7\n",
+            "155                    0                    0\n",
+            "156                   14                   21\n",
+            "157                   18                   31\n",
+            "158                    8                   14\n",
+            "159                    0                    0\n",
+            "160                    0                    0\n",
+            "161                    4                    3\n",
+            "162                    0                    0\n",
+            "163                    0                    0\n",
+            "164                    0                    0\n",
+            "165                    0                    0\n",
+            "166                    0                    0\n",
+            "167                    8                    7\n",
+            "168                    3                    7\n",
+            "169                   12                   16\n",
+            "170          
```

Comment:
Can we do a plot or something to show some indication of similarity? Or at least dump an interactive table with data cols and sampled cols? Note that colab will show an interactive table if the final line of the cell produces a pd.DataFrame